README.md

```diff
@@ -57,3 +58,35 @@ xattr -dr com.apple.quarantine manticore-executor
 ### Windows support
 
 We do not support Windows for now but this work is in progress. For now you can just install PHP 8.1.10+ and enable the requested extensions (see above).
+
+## Deployment
+
+1. [The GitHub actions workflow](.github/workflows/release.yml) prepares packages/binaries for
+
+   * Centos 7/8 x86_64 and arm64
+   * Ubuntu Xenial/Bionic/Focal/Jammy x86_64 and arm64
+   * Debian Stretch/Buster/Bullseye x86_64 and arm64
+   * MacOS x86_64
+
+2. The same workflow deploys the packages to [repositories](https://repo.manticoresearch.com)
+
+3. What's to be done manually after the above is:
+
+   * build binaries for MacOS arm64 following [this instruction](#how-to-build-manually)
```

Comment:
wrong link?

Reply:
The link points to self documentation – it's correct